Sources/Generics/GenericTableViewController.swift

```diff
@@ -205,7 +205,7 @@ private extension GenericTableViewController {
     }
 
     func setupTable() {
-        tableView = UITableView(frame: .zero, style: .grouped)
+        tableView = UITableView(frame: .zero, style: .plain)
```

Comment:
update this to 
``` if #available(iOS 11.0, *) {
            tableView = UITableView(frame: .zero, style: .grouped)
            tableView.tableFooterView = UIView()
        } else {
            tableView = UITableView(frame: .zero, style: .plain)
            let footerView = UIView(frame: CGRect(x: 0, y: 0, width: tableView.frame.size.width, height: 0.5))
            footerView.backgroundColor = tableView.separatorColor
            tableView.tableFooterView = footerView
        }
```

Reply:
@jsaini-hw is right. We already had similar workaround in SelectTransferMethodTypeTableViewController.swift, setupTransferMethodTypeTableView()

Reply:
@jsaini-hw, @syyang-hw 
Thanks, changed.

Btw, in my branch at least `setupTransferMethodTypeTableView()` doesn't have this workaround.
Should I add it there as well?